app/src/main/res/layout/hints_and_solution_fragment.xml

```diff
@@ -33,24 +44,23 @@
       app:layout_constraintEnd_toEndOf="parent"
       app:layout_constraintStart_toStartOf="parent"
       android:background="@color/hintsAndSolutionBackground"
-      app:layout_constraintTop_toBottomOf="@+id/hints_and_solution_toolbar">
+      app:layout_constraintTop_toBottomOf="@+id/hints_and_solution_app_bar_layout">
 
       <androidx.recyclerview.widget.RecyclerView
         android:id="@+id/hints_and_solution_recycler_view"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:clipToPadding="false"
         android:overScrollMode="never"
-        android:paddingTop="8dp"
-        android:paddingBottom="128dp"
+        android:paddingTop="20dp"
+        android:paddingBottom="168dp"
```

Comment:
This is inconsistent in the mocks meaning, it is `152dp` in mock but I remember in various other mocks it is `168dp`. So you can finalise this what should it be finalise my suggestion is `168dp` to keep it consistent.

Reply:
Okay, keeping it 168dp.